ludwig/globals.py

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ==============================================================================
 
-LUDWIG_VERSION = "0.10.dev"
+LUDWIG_VERSION = "0.10.1.dev"
```

Comment:
Should this be 0.10.2.dev since we're developing 0.10.2?

Reply:
@arnavgarg1 This is a very good question.  To me, the future version is not know (i.e., we could be developing 1.0).  Hence, my approach was in the style of "current_version_+" -- meaning that "what we are doing on top of the currently shipping version".  What do you think?  Thank you.

Reply:
Sounds good to me!